### ISSUE TITLE
[ask] How to get Mysql position?

### ISSUE THREAD

**Issue by yeka** [go-mysql-org/go-mysql#106]
In previous version, we have syncer.ExecuteSql("SHOW MASTER STATUS") to get Mysql position.
How to do that now?

**Comment by siddontang** [comment]
Hi @yeka 

You can still use this command, get the position and pass to BinlogSyncer now.

**Comment by yeka** [comment]
Hi @siddontang 

I couldn't find `ExecuteSql()` method under BinlogSyncer. Or should I create 2 connections now (Client and Replication)?

**Comment by siddontang** [comment]
@yeka 

Yes, now I want BinlogSyncer do only do one thing: sync, so I remove ExecuteSql. You can use another connection to get the position first like canal does.

**Comment by 3manuek** [comment]
@yeka I wrapped the _show master status_ like this to make my life easier. `execute` is used from `client` library:

```
func getCoordinates(conn *client.Conn) (p mysql.Position) {
	r, _ := conn.Execute("SHOW MASTER STATUS")
	binFile, _ := r.GetString(0, 0)
	binPos, _ := r.GetInt(0, 1)
	p.Name = binFile
	p.Pos = uint32(binPos)
	return p
}
```
